src/coreclr/vm/exceptionhandling.h

```diff
@@ -673,10 +563,6 @@ private: ;
 
     void ReleaseResources();
 
-    void SetEnclosingClauseInfo(bool     fEnclosingClauseIsFunclet,
-                                DWORD    dwEnclosingClauseOffset,
-                                UINT_PTR uEnclosingClauseCallerSP);
-
     struct EnclosingClauseInfo
```

Comment:
This should not be needed

Reply:
There's a rather long usage chain for this going from `StackFrameIterator::Filter` -> `GetCallerOfEnclosingClause`.